test/features/perform_sanity_checks_on_reviews_test.rb

```diff
@@ -0,0 +1,194 @@
+require 'test_helper'
+Dir[Rails.root.join('test/**/*.rb')].each { |f| require f }
+
+class PerformsSanityChecksOnReviewsTest < Capybara::Rails::TestCase
+  include Helpers::UserHelpers
+
+  before do
+    stub_request(:get, "#{Cmr.get_cmr_base_url}/search/granules.echo10?concept_id=G309210-GHRC")
+      .with(
+        headers: {'Accept-Encoding' => 'gzip;q=1.0,deflate;q=0.6,identity;q=0.3', 'Accept' => '*/*', 'User-Agent' => 'Ruby'}
+      )
+      .to_return(status: 200, body: '<?xml version="1.0" encoding="UTF-8"?><results><hits>0</hits><took>32</took></results>', headers: {})
+  end
+
+  describe 'performs "mark complete" on a collection record' do
+    before do
+      OmniAuth.config.test_mode = true
+      mock_login(role: 'arc_curator')
+    end
+
+    # in this case short name, long name are marked with a color, but version id is not
+    it 'will perform sanity checks on BOTH collection and assoc granule review' do
```

Comment:
I am concerned it might be difficult to debug this test when the workflow breaks.  Breaking it into smaller tests may make it more digestible.

Reply:
I agree, there is no helpers for tests, feel like this should be broken up into a few helpers as well.